googleCalender.js

```diff
@@ -0,0 +1,447 @@
+const max = 50;
+async function UsersFullSync({ dataStore, client }) {
+  console.log(" sync started");
+  const responsedata = await client.fetch("admin/directory/v1/users?customer=my_customer&projection=full&viewType=admin_view");
+  //console.log(JSON.stringify(responsedata));
+  if (!responsedata.ok) {
+    throw new Error(
+      `Accounts sync failed ${responsedata.status}:${responsedata.statusText}.`
+    );
+  }
+  let data = await responsedata.json();
```

Comment:
Use let only when the value is not constant. 
```suggestion
  const data = await responsedata.json();
```